JULIERAJ/KIDS-FIRST-Front
# Fix/121/sidebar header active state

## One Line Description
Sidebar and Header does not change active state correctly

## Test Steps
1. Open the application.
2. Navigate to the sidebar and click on the "Home" button.
3. Click on the "Add Your Kid +" button. The kids' info will appear in the header and be highlighted in the sidebar.

## Checklist
